src/DynamoCore/Properties/AssemblyInfo.cs

```diff
@@ -58,6 +58,8 @@
 [assembly: InternalsVisibleTo("DocumentationBrowserViewExtension")]
 [assembly: InternalsVisibleTo("Notifications")]
 [assembly: InternalsVisibleTo("NodeAutoCompleteViewExtension")]
+[assembly: InternalsVisibleTo("MCPExtension")]
+[assembly: InternalsVisibleTo("MCPViewExtension")]
```

Comment:
Do we have both of these extensions today or is it future proofing for a case where we may have an additional one in the near future?

Reply:
Future proofing. Today it is a ViewExtension, it could be just an extension in the future.